src/components/pages/search/Results/ResultsList/Result/ResultContainer.js

```diff
@@ -0,0 +1,16 @@
+import { connect } from 'react-redux'
+import Result from './Result'
+import { selectSearchGroupBySearchResult } from '../../../../../../redux/selectors/data/categoriesSelectors'
+
+const mapStateToProps = (state, ownProps) => {
+  const { result } = ownProps
+  const { offer } = result
+  const searchGroup = selectSearchGroupBySearchResult(state, offer)
+  const searchGroupLabel = searchGroup ? searchGroup.value : ''
```

Comment:
Puisqu'on définit une valeur par défaut à la prop `searchGroupLabel` dans le composant Result, on pourrait se permettre d'avoir un searchGroupLabel undefined et faire quelque chose du genre
```suggestion
  const searchGroupLabel = searchGroup?.value
```
À toi de voir si tu préfères conserver ceinture-bretelles ;)

Reply:
:+1: